content/300-guides/050-database/100-developing-with-prisma-migrate/200-troubleshooting-development.mdx

```diff
@@ -5,18 +5,18 @@ metaDescription: Troubleshooting issues with Prisma Migrate in a development env
 
 <TopBlock>
 
-<Admonition type="warning">
-
-**MongoDB not supported** <br />
-Prisma Migrate does not currently support the [MongoDB connector](/concepts/database-connectors/mongodb).
-
-</Admonition>
-
 This guide describes how to resolve issues with Prisma Migrate in a development environment, which often involves resetting your database. For production-focused troubleshooting, see:
 
 - [Production troubleshooting](/guides/database/production-troubleshooting)
 - [Patching / hotfixing production databases](/guides/database/patching-production)
 
+<Admonition type="info">
+
+This guide **does not apply for MongoDB**.<br />
+Instead of `migrate dev`, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
+
+</Admonition>
+
```

Comment:
Point out that Migrate for MongoDB is db push instead of "not supported" warning.